media/css/firefox/firstrun/nightly.scss

```
@@ -82,3 +74,12 @@ main {
         @include text-title-sm;
     }
 }
+
+.mzp-l-columns .mzp-c-picto {
+    margin-bottom: $layout-md;
+    @media #{$mq-md} {
+        margin-bottom: 0;
+        padding: 48px;
+        padding-bottom: 0;
```

We can combine this with the rule above and use a token for the value:
```suggestion
        padding: $spacing-2xl $spacing-2xl 0;
```